superset-frontend/src/CRUD/Field.jsx

```diff
@@ -32,7 +32,7 @@ import './crud.less';
 const propTypes = {
   value: PropTypes.any.isRequired,
   label: PropTypes.string.isRequired,
-  descr: PropTypes.node,
```

Comment:
why make this shorter and less clear? Turns out because this file is JSX, some usages of this component set the `description` prop, which doesn't even work!

I've cleaned it all up to use `description` instead